tests/unittest/_torch/ray_orchestrator/multi_gpu/test_llm_update_weights_multi_gpu.py

```
@@ -761,32 +768,80 @@ def filter_fn(name: str) -> bool:
+        "--run-ray",
+        "-p",
+        "no:cacheprovider",
+        "-p",
+        "no:xdist",
+        "--tb=short",
+        "-s",
+        "-v",
+    ]
+    env = {**os.environ, _NEMOTRON_H_BODY_ENV: "1"}
+    try:
+        result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)